Documentation/helm-operator.md

```diff
@@ -58,16 +69,21 @@ To install the helm chart from master, you will need to pass the specific versio
 
 ```console
 helm repo add rook-master https://charts.rook.io/master
+# For Helm v3.x
+helm search repo rook-ceph --versions
+# For Helm v2.x
 helm search rook-ceph
-helm install --namespace rook-ceph rook-master/rook-ceph --version <version>
+helm install --namespace rook-ceph rook-ceph rook-master/rook-ceph --version <version>
 ```
 
-For example:
+For example to install version `v0.7.0-278.gcbd9726`:
 
 ```cosnole
```

Comment:
Can you fix this while we are at it? `s/cosnole/console/`

Reply:
@leseb Fixed, PTAL